lib/screens/rating_page.dart

```diff
@@ -0,0 +1,166 @@
+              children: <Widget>[
+                SizedBox(
+                  height: 40.0,
+                ),
+                Center(
+                  child: Column(
+                    children: [
+                      _profilePicture('assets/images/batman.png'),
+                      _heading('How was your session with', 24.0),
+                      _heading('Tony N?', 24.0),
+                      _ratingBar(_ratingBarMode),
+                    ],
+                  ),
+                ),
+                SizedBox(
+                  height: 20.0,
+                ),
+                _rating != null
+                    ? Text(
+                        'Rating: $_rating',
+                        style: TextStyle(fontWeight: FontWeight.bold),
+                      )
+                    : Container(),
+                SizedBox(
+                  height: 40.0,
+                ),
+                Center(
+                  child: _submitButton('Done!'),
+                ),
+              ],
+            ),
+          )),
+        ),
+      ),
+    );
+  }
+
+  void checkRating() {
+    double checker = _rating;
+    if (checker != null) {
+      print(checker);
+    } else {
+      _rating = 0;
+      print(_rating);
+    }
+  }
+
+  Widget _profilePicture(String profileImage) {
+    return Center(
+      child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            CircleAvatar(
+              radius: 55.0,
+              backgroundColor: Colors.black,
+              child: CircleAvatar(
+                radius: 50.0,
+                backgroundImage: AssetImage(profileImage),
+              ),
+            ),
+          ]),
+    );
+  }
+
+  Widget _heading(String text, double fontSize) => Container(
+        child: Column(
+          children: [
+            Text(
+              text,
+              style: TextStyle(
+                color: Color(0xFF666666),
+                fontWeight: FontWeight.w300,
+                fontSize: 24.0,
+              ),
+            ),
+            SizedBox(
+              height: 20.0,
+            ),
+          ],
+        ),
+      );
+  Widget _submitButton(String text) => Column(
+        children: [
+          ElevatedButton(
+            onPressed: () async {
+              checkRating();
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) {
+                    return HomePage();
+                  },
+                ),
+              );
+            },
+            child: Text(
+              text,
+              style: TextStyle(
+                color: Colors.white,
+                fontWeight: FontWeight.w300,
+                fontSize: 24.0,
+              ),
+            ),
+          ),
+        ],
+      );
+
+  Widget _ratingBar(int mode) {
+    return RatingBar.builder(
```

Comment:
Can you add an argument of `glow: false,` ? Just a styling change, I think the glow when dragging is tacky.